src/core/nconn/nconn.cc

```diff
@@ -245,7 +245,7 @@ int32_t nconn::nc_set_listening_nb(int32_t a_val)
 int32_t nconn::nc_set_accepting(int a_fd)
 {
         int32_t l_s;
-        l_s = ncset_accepting(a_fd);
+        l_s = ncset_accepting(a_fd); //function name typo?
```

Comment:
This comment was a little cryptic - sorry about that.
I flagged it because it looked to me like a call to the `nconn::nc_set_accepting` function, which wouldn't make sense since it would have been a call to the function in which it is called from.

After looking at it again, I realised now that there is a difference between `ncset_accepting` (which is declared as a virtual function in nconn.h, and subsequently used/defined in the derivative classes nconn_tcp and nconn_tls) and `nc_set_accepting` (which is declared in nconn.h and defined in nconn.cc).

This code isn't particularly relevant to the KTLS changes in this PR, but I had a look at it while doing a 'sanity check' to determine if/when the `nconn_tls::ncset_accepting` function is ever called/used by hurl, since the call to `SSL_set_fd` within this function will destroy (free) any BIOs attached to m_tls.